README.md

```diff
@@ -1,24 +1,30 @@
-# Example ComputerCraft datapack
-This is an exmaple datapack for ComputerCaft, which can be used as a skeleton
+# Example ComputerCraft Datapack
+This is an example datapack for ComputerCraft, which can be used as a skeleton
 for your own pack.
 
 Datapacks can be used to add or overwrite files in [ComputerCraft's own rom][rom],
-allowing you to add new programs or APIs, or adjust the behaviour of existing
+allowing you to add new programs or modules, or adjust the behavior of existing
 ones. This pack comes with several features:
 
- - `apis/my_api.lua`: A new API with a single method `my_api.do_something()`.
- - `programs/my-program.lua`: A new program `my-program`, which just calls the
-   above function.
+ - `modules/main/my_module.lua`: A new module with a single method `my_api.do_something()`.
+ - `programs/my_program.lua`: A new program `my_program`, which just calls `do_something` from above.
+ - `modules/turtle/helpers.lua`: A new module with some movement aliases and a `digColumn` method.
+ - `programs/turtle/demo.lua`: A new turtle program that uses the `helpers` module to dig 3x3 openings.
  - `autorun/my-pack.lua`: A basic program which is run whenever a computer turns
    on. This may be used to perform additional setup, such as [registering
    autocompletion][completion] for your programs.
 
-## Using the pack
-This should be placed in the "datapacks" folder of your world. The resulting file structure should look something like this:
+(Note: files in `turtle` folders will only be visible to turtles!)
+
+## Using the Pack
+The contents of this project should be zipped<sup>1</sup> and placed in the `datapacks`
+folder of your world's save folder.
+The resulting file structure should resemble the the built-in ROM folder,
+looking something like this:
 
 ```
 datapacks
-└─my-pack
+└─my-pack.zip
```

Comment:
I think it might be worth putting `my-pack(.zip)` here, and still mentioning that you can have a normal directory. It's obviously not as good for distribution, but makes developing much easier.

Reply:
When I tried with a plain dir it didn't work? Maybe I did something that broke it, but being able to edit in-place without re-zipping would be nice.